src/jace/translator/primitive_translators/alu_translator.py

```diff
@@ -0,0 +1,292 @@
+        if is_scalar:
+            tskl_output = ("__out0", dace.Memlet.simple(out_var_names[0], "0"))
+        else:
+            tskl_output = (
+                "__out0",
+                dace.Memlet.simple(out_var_names[0], ", ".join([X[0] for X in tskl_map_ranges])),
+            )
+
+        if is_scalar:
+            tskl_tasklet = eqn_state.add_tasklet(
+                tskl_name,
+                _list_to_dict(tskl_inputs).keys(),
+                _list_to_dict([tskl_output]).keys(),
+                tskl_code,
+            )
+            for in_var, (in_connector, in_memlet) in zip(in_var_names, tskl_inputs, strict=False):
+                if in_var is None:  # So access node for literal
+                    continue
+                eqn_state.add_edge(
+                    eqn_state.add_read(in_var),
+                    None,
+                    tskl_tasklet,
+                    in_connector,
+                    in_memlet,
+                )
+            eqn_state.add_edge(
+                tskl_tasklet,
+                tskl_output[0],
+                eqn_state.add_write(out_var_names[0]),
+                None,
+                tskl_output[1],
+            )
+        else:
+            eqn_state.add_mapped_tasklet(
+                name=tskl_name,
+                map_ranges=_list_to_dict(tskl_map_ranges),
+                inputs=_list_to_dict(tskl_inputs),
+                code=tskl_code,
+                outputs=_list_to_dict([tskl_output]),
+                external_edges=True,
+            )
+
+        return eqn_state
+
+    def _write_tasklet_code(
+        self,
+        in_var_names: Sequence[str | None],
+        eqn: jax_core.JaxprEqn,
+    ) -> str:
+        """This function generates the Tasklet code based on a primitive.
+
+        The function will also perform literal substitution and parameter handling.
+
+        Args:
+            in_var_names:   The list of SDFG variables used as input.
+        """
+
+        t_code = self._prim_tmpl
+
+        # Now we handle Literal substitution
+        for i, in_var_name in enumerate(in_var_names):
+            if in_var_name is not None:
+                continue
+
+            jax_in_var: jax_core.Literal = cast(jax_core.Literal, eqn.invars[i])
+            if util.get_jax_var_shape(jax_in_var) == ():
+                t_val = jax_in_var.val
+                if isinstance(t_val, np.ndarray):
+                    t_val = jax_in_var.val.max()  # I do not know a better way in that case
+                t_code = t_code.replace(f"__in{i}", str(t_val))
+            else:
+                raise ValueError(
+                    f"Can not handle the literal case of shape: {util.get_jax_var_shape(jax_in_var)}"
+                )
+
+        # Now replace the parameters
+        if len(eqn.params) != 0:
+            t_code = t_code.format(**eqn.params)
+
+        return t_code
+
+
+def _list_to_dict(inp: Sequence[tuple[None | Any, Any]]) -> dict[Any, Any]:
```

Comment:
Just `list_to_dict()` is misleading. I would make it more explicit if possible, like:
```suggestion
def _list_to_filtered_dict(inp: Sequence[tuple[None | Any, Any]]) -> dict[Any, Any]:
```